constance/utils.py

```diff
@@ -1,6 +1,27 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
 from importlib import import_module
 
+from constance import LazyConfig, settings
+
+config = LazyConfig()
```

Comment:
I was having some problems with circular imports, I'll see if I can tidy this up